client/src/main/com/sinch/sdk/domains/verification/models/requests/StartVerificationRequestParameters.java

```diff
@@ -117,9 +116,9 @@ public static class Builder<B extends Builder<B>> {
      * @param method The type value
      * @since 1.0
      */
-    protected Builder(VerificationMethodType method) {
-      this.method = OptionalValue.of(method);
-    }
+    /*protected Builder(VerificationMethodType method) {
```

Comment:
You can remove the commented methods and their bodies when abstract

Reply:
Removed